utilities/review/constants/utilitary.ts

```diff
@@ -1,7 +1,23 @@
+import { isHex } from "viem";
+import { Hex } from "viem";
+import { registryHelper, ThemeMode } from "@/components/Pages/ProgramRegistry/helper";
+
 export const addPrefixToIPFSLink = (link: string) => {
   if (link.startsWith("ipfs://")) {
     return link.replace("ipfs://", "https://ipfs.io/ipfs/");
   } else {
     return link;
   }
 };
+
+export function validateHex(potentialHex: string | Hex): Hex {
+  if (isHex(potentialHex)) {
+    return potentialHex as Hex;
+  } else {
+    throw new TypeError(`Invalid hex value: ${potentialHex}`);
+  }
+}
+
+export function getNetworkImage(network: string, themeMode: ThemeMode) {
```

Comment:
If this did not include a Theme, it could be reused in many other places as well ;)

Reply:
Do you think I should change that to something else?

Reply:
Yes! Remove the theme spec and reuse it whenever the rest of the code is referenced: you will see that multiple things places more will use it